R/tbl-format.R

```diff
@@ -60,17 +60,39 @@ format_tbl <- function(
   force(x)
   num_colors(forget = TRUE)
 
-  setup <- tbl_format_setup(x,
-    width = width, ...,
+  setup_used <- FALSE
+
+  setup <- tbl_format_setup(
+    x,
+    width = width,
+    ...,
+    setup = {
+      setup_used <- TRUE
```

Comment:
I'd add a note in here about the fact that "If `setup` is evaluated by the `tbl_format_setup()` method, then that's our signal to recall `tbl_format_setup()` a second time"